var.mk

```diff
@@ -1,5 +1,10 @@
 # to be included by Makefile
 
+STACK_EXISTS := $(shell command -v stack 2> /dev/null)
+ifndef STACK_EXISTS
+    $(error "stack is not available please install haskell-stack")
+endif
+
```

Comment:
I don't understand it.

> As said: please make sure, that stack is there before using it blindly!

Here you say that stack needs to be there.

> There is no need, that stack exists

Here you say, it doesn't need to be there.

I understand that it doesn't need to be available for clean targets etc, but the `var.mk` uses it a lot because the "what version of package x is installed" checks *need* to use stack. And these are done for *every* target. So actually, stack needs to be there for clean targets.